tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java

```diff
@@ -99,20 +101,16 @@ public class ShowLocaleCoverage {
 
     private static final String PROPERTIES_HEADER =
         // Extra tabs are for github table formatting
-        "#\tcoverageLevels.txt\t\n"
-        + "#\tCopyright © 2022 Unicode, Inc.\n"
-        + "#\tCLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)\n"
-        + "#\tFor terms of use, see http://www.unicode.org/copyright.html\n"
-        + "#\t\n"
-        + "#\tProvides the Coverage Level of locales at Basic or Above.\n"
-        + "#\tFor more info,see the Locale Coverage Chart for this version.\n"
-        + "#\tGenerated by ShowLocaleCoverage.\n"
-        + "#\t\n"
-        + "#Locale"
-        + " ;\tLevel"
-//            + " ;\t% of Higher"
-//            + " ;\tHigher Level"
-;
+        "# coverageLevels.txt\n"
+        + "# Copyright © 2022 Unicode, Inc.\n"
+        + "# CLDR data files are interpreted according to the\n"
+        + "# LDML specification: http://unicode.org/reports/tr35/\n"
+        + "# For terms of use, see http://www.unicode.org/copyright.html\n"
+        + "#\n"
+        + "# For format and usage information, see:\n"
+        + "# https://cldr.unicode.org/index/cldr-spec/coverage-levels.\n"
```

Comment:
```suggestion
        + "# For format and usage information, see:\n"
        + "# https://cldr.unicode.org/index/cldr-spec/coverage-levels\n"
```

drop the trailing `.`